apps/site/src/views/Home/sections/Sponsors/Sponsors.tsx

```diff
@@ -1,13 +1,32 @@
-"use client";
+/* eslint-disable @next/next/no-img-element */
+import { getSponsors } from "./getSponsors";
+import styles from "./Sponsors.module.scss";
+import { client } from "@/lib/sanity/client";
+import imageUrlBuilder from "@sanity/image-url";
+import clip from "./clip.svg";
 
-import Container from "react-bootstrap/Container";
+const builder = imageUrlBuilder(client);
+
+export default async function Sponsors() {
+	const sponsors = await getSponsors();
 
-function Sponsors() {
 	return (
-		<Container as="section">
-			<h2>Sponsors</h2>
-		</Container>
+		<section className={styles.container}>
+			<div className={styles.clipboard}>
+				<img className={styles.clip} src={clip.src} alt="" />
+				<h1 className={styles.title}>Sponsors</h1>
```

Comment:
Issue: the Home page title is used for heading level 1. For this section heading, level 2 would be more appropriate.